.github/workflows/build_and_test.yml

```diff
@@ -127,3 +144,9 @@ jobs:
 
       - name: Build Android Client
         run: npx gulp build --platform=android
+
+      - name: Upload Android Client
+        uses: actions/upload-artifact@v2
+        with:
+          name: unsigned-apk
+          path: platforms/android/app/build/outputs/apk
```

Comment:
Isn't the file `Outline.apk`?

Reply:
What's in the folder?
These are the filenames I see in outline-releases: https://github.com/Jigsaw-Code/outline-releases/tree/master/client

Though we may not be outputting the same thing here.

Reply:
It seems there are multiple files there: https://github.com/Jigsaw-Code/outline-client/runs/4445892346?check_suite_focus=true#step:8:57

Can we upload multiple files too?

Reply:
Ignore this. The directory spec will make all files to be added to the artifact zip file, so we're good.